clusterloader2/testing/density/constraints_100.yaml

```diff
@@ -0,0 +1,24 @@
+coredns:
+  cpuConstraint: 0.01
+  memoryConstraint: 9437184 #9 * (1024 * 1024)
```

Comment:
Some constrantings seem to be pretty aggresive here - e.g. this 9MB.
I think with such constraints we will see flake sooner or later.

Setting memory to less than 20-30MB I think is too aggressive.